src/components/FileList/__tests__/__snapshots__/FileElement.test.js.snap

```diff
@@ -125,7 +125,7 @@ exports[`renders correctly 1`] = `
               datetime="2018-09-19T08:33:31.000Z"
               title="2018-09-19T08:33:31+00:00"
             >
-              7 months ago
+              8 months ago
```

Comment:
The data mock method for snapshot test was updated in pr #223, if you rebase master and regenerate the snapshot test, this should go away.

Reply:
updated and amended to last commit